samples/dotnet-azurefunction/README.md

```diff
@@ -390,24 +390,8 @@ kind: Secret
 Now you should have all Dapr components up and running in your kubernetes cluster. Next we will show how to deploy your function app into your kubernetes cluster with the Dapr Side Car.
 
 ## Deploy your Dapr Function App
-You can find your function app deployment file `deploy/function.yaml`. Let's take a look:
+You can find your function app deployment file `deploy/functionapp.yaml`. Let's take a look:
 
```

Comment:
nit: remove this "let's take a look" as we did not show the yaml snippet anymore